PTGLdynamics/codes/plotting-scripts/PyMolHeatmapVisualisation.py

```diff
@@ -172,6 +178,27 @@ def sorted_nicely( l ):
                        default = '',
                        help = 'specify a path to your output files. Otherwise the current folder is used.')
 
+cl_parser.add_argument('--divide-by-chainlength',
+                       metavar = 'divide-by-chainlength',
```

Comment:
Metavar defines a placeholder the help text uses. In this case `chainlength` would be clearer. EDIT: After reading the help text, I suggest something that contains "path" or "file". Ideally, the metavar gives a good impression on what is expected. In such as case, "PATH" would be sufficient. Additional information is given by the help text.